CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
+-   Added `Disabled` prop to `Button` component and removed Disabled state
```

Comment:
This should mention the removal of the `state` property and the addition of the disabled and loading states onto the api